internal/handlers/sqlite/cmd_query.go

```diff
@@ -31,23 +32,38 @@ func (h *Handler) CmdQuery(ctx context.Context, query *wire.OpQuery) (*wire.OpRe
 	cmd := query.Query.Command()
 	collection := query.FullCollectionName
 
-	// both are valid
-	if (cmd == "ismaster" || cmd == "isMaster") && collection == "admin.$cmd" {
-		return &wire.OpReply{
-			NumberReturned: 1,
-			Documents: []*types.Document{must.NotFail(types.NewDocument(
-				"ismaster", true, // only lowercase
-				"maxBsonObjectSize", int32(types.MaxDocumentLen),
-				"maxMessageSizeBytes", int32(wire.MaxMsgLen),
-				"maxWriteBatchSize", int32(100000),
-				"localTime", time.Now(),
-				"connectionId", int32(42),
-				"minWireVersion", common.MinWireVersion,
-				"maxWireVersion", common.MaxWireVersion,
-				"readOnly", false,
-				"ok", float64(1),
-			))},
-		}, nil
+	if strings.Contains(collection, "$cmd") {
```

Comment:
Same comment as pg. Also we prefer to move duplicated code to common handler instead of copy and paste :see_no_evil: